queries/python.scm

```diff
@@ -0,0 +1,64 @@
+;; Generated by the following command:
+;; > curl https://raw.githubusercontent.com/tree-sitter/tree-sitter-python/d6210ceab11e8d812d4ab59c07c81458ec6e5184/src/node-types.json \
+;;   | jq '[.[] | select(.type == "_simple_statement" or .type == "_compound_statement") | .subtypes[].type]'
+[
+  (assert_statement)
+  (break_statement)
+  (class_definition)
+  (continue_statement)
+  (decorated_definition)
+  (delete_statement)
+  (exec_statement)
+  (expression_statement)
+  (for_statement)
+  (function_definition)
+  (future_import_statement)
+  (global_statement)
+  (if_statement)
+  (import_from_statement)
+  (import_statement)
+  (nonlocal_statement)
+  (pass_statement)
+  (print_statement)
+  (raise_statement)
+  (return_statement)
+  (try_statement)
+  (while_statement)
+  (with_statement)
+] @statement
+
+(
+  (function_definition
+    name: (_) @functionName
+    body: (_) @namedFunction.interior
+  ) @namedFunction @functionName.domain
```

Comment:
we are able to pack a lot of information in here.  note how we've defined "funk", "inside funk", and "funk name" all in one go